sunpy/net/vso/attrs.py

```diff
@@ -18,7 +18,6 @@
 import collections
 
 import astropy.units as u
-from astropy.time import Time
```

Comment:
So this change I'm unsure about. Within the `Time` attr, the method `pad` returns a `Time`but I assume this must be a new `Time` attr and not astropy.time.Time instance. If that is the case, then we don't need this import.

Reply:
This import would get clobbered by the `Time` attr anyway right? I think you should be good here.

Reply:
It's what the linter said should happen.